### PR TITLE
Updated DreamSpark to MS Imagine

### PR DESCRIPTION
**Pull request by bithakr**
per recent rebranding + add details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ Table of Contents
 
 ## Other Packs
 
-  * [Microsoft DreamSpark](https://www.dreamspark.com/Student/Default.aspx) - DreamSpark is a Microsoft Program that supports technical education by providing access to Microsoft software for learning, teaching and research purposes.
+  * [Microsoft DreamSpark](https://imagine.microsoft.com/) - Imagin is a Microsoft Program that supports technical education by providing access to Microsoft software for learning, teaching and research purposes. Currently includes SQL Server, Windows Server 2008-12, and Windows 8 Embedded Industry Pro. Most students are also eligible for a free Windows 10 license throught [OnTheHub](http://onthehub.com/download/free-software/windows-10-education-for-students/).
```
Comment (keplersj) [inline README.md]: Shouldn't the list item be updated too? I think `Imagin` might be a typo.

**Comment by AchoArnold** [comment]
ping @bithakr

**Comment by bithakr** [comment]
oops... missed that. will fix now

**Comment by AchoArnold** [comment]
@bithakr You still have a typo. You have a typo here. It should be **through** instead of **throught**

**Comment by AchoArnold** [comment]
Thanks @bithakr

**Pull request merged**